content/blog/2022-2023/2022-06-05-ccss-summer-dev-projects-introduction.md

```diff
@@ -31,3 +31,11 @@ part of the Rust Gamedev Working Group, and the team lead of the Rust CTCFT
 team. I’m also one of the core developers on Veloren, an open source game
 similar to Minecraft. In the Rust communty, I run a monthly game dev meetup, and
 co-edit the monthly game dev newsletter.
+
+### Victor Li ([@VictorLi5611](https://github.com/VictorLi5611))
+
+Hey, y'all!! My name is Victor. 
+
+Currently, I'm a 3rd year CS student w/ honors in the AI stream at Carleton University in Ottawa, Canada. I'm also the Director of Finance or Treasurer of the Carleton Computer Science Society, where I manage our finances and merch. I'm also from Toronto and moved to Ottawa to study at Carleton!! I like to do in my spare time to read manga, watch anime, and playing some chess. If it's sunny out, I also kayak and bike around Ottawa. 
```

Comment:
What is your favorite anime? Mine is Gurren Lagann.

Reply:
My favourite currently anime is either Your Name or Assassination Classroom.